src/components/NcActionButton/NcActionButton.vue

```diff
@@ -409,11 +413,12 @@ export default {
 		},
 
 		/**
-		 * If this is a menu, a chevron icon will
-		 * be added at the end of the line
+		 * Weather the button opens a submenu
```

Comment:
```suggestion
		 * Wether the button opens a submenu
```
:3

Reply:
🌦️

Reply:
merde, that's how you know I was using my phone :p

Reply:
Fixed